infrastructure/merklemountainrange/src/merkle_change_tracker.rs

```diff
@@ -49,6 +52,12 @@ pub(crate) struct MerkleCheckPoint {
     pub objects_to_del: Vec<ObjectHash>,
     mmr_to_add: Vec<MerkleNode>,
 }
+
+pub(crate) struct CpCleanup {
```

Comment:
```suggestion
pub(crate) struct CheckpointCleanup {
```

Writing MMR out would be a screenful, Checkpoint is fine